test/hotspot/jtreg/serviceability/sa/TestHeapDumpForInvokeDynamic.java

```diff
@@ -37,6 +37,7 @@
 import jdk.test.lib.Utils;
 import jdk.test.lib.hprof.parser.HprofReader;
 import jdk.test.lib.hprof.parser.PositionDataInputStream;
+import jdk.test.lib.hprof.HprofParser;
```

Comment:
PositionDataInputStream and HprofParser imports are not used anymore

Reply:
Thanks for review, all copyright year and unused imports are removed now